src/bioregistry/data/bioregistry.json

```diff
@@ -102567,6 +102567,34 @@
       }
     ]
   },
+  "rnaprodb": {
+    "contact": {
+      "email": "rohs@usc.edu",
+      "name": "Remo Rohs",
+      "orcid": "0000-0003-1752-1884"
+    },
+    "contributor": {
+      "email": "palayoor.n@northeastern.edu",
+      "github": "nalikapalayoor",
+      "name": "Nalika Palayoor",
+      "orcid": "0009-0008-8406-631X"
+    },
+    "description": "RNAproDB is an interactive dashboard designed to assist research in structural analysis of RNA-protein conplexes. For each complex, the dashboard gives information about the nucleic acid structure as well as interacting proteins.",
```

Comment:
It's not clear from this description what the identifiers in this resource are. Please update the description to first describe what entities this semantic space is about, then after, it's okay to give a more general description of the resource.

Please also take a look at the guidelines on writing a good description: https://github.com/biopragmatics/bioregistry/blob/main/docs/CONTRIBUTING.md#writing-a-good-description